go/vm/geth/geth.go

```diff
@@ -126,6 +137,11 @@ func createGethInterpreterContext(parameters vm.Parameters) (*geth.EVM, *geth.Co
 		Transfer:    transferFunc,
 		CanTransfer: canTransferFunc,
 	}
+
+	if parameters.Revision >= vm.R11_Paris {
+		blockCtx.Random = &common.Hash{}
```

Comment:
This should be commented with something like:
```suggestion
                // Setting the random signals to geth that a post-merge (Paris) revision should be utilized.
		blockCtx.Random = &common.Hash{}
```